### ISSUE TITLE
eyeStateDetector.IsOpen 

### ISSUE THREAD

**Issue by zoekatakuzinos** [georg-jung/FaceAiSharp#57]
Thank you Georg for a great project! 

I am calling eyeStateDetector.IsOpen(faceImage) on various of my images as I am looking into adding liveness checks. Ultimately I will be detecting and recognizing frames from a webcam. I'm experimenting with the possibility of looking for blinking.

The call to eyeStateDetector.IsOpen(faceImage) seems to return true even on an image where I have closed my eyes, and then calls false on an image where my head is tilted and looking away with eyes open. I am not sure why? 
When I ran through your demo on the website and tried your "count open eyes" it worked with those same images so I am not sure what I am doing wrong. I am not cropping the image with only the eye part when I call eyeStateDetector.IsOpen(faceImage). 

Could that be the problem? 

Thank you

**Comment by georg-jung** [comment]
Yes, the eyeStateDetector.IsOpen method does expect an image that mostly contains only the eye. Take a look at <https://github.com/georg-jung/FaceAiSharp/blob/aabb31d0bda61bbb1aa234c182f900f6e30c3dfe/src/FaceAiSharp/Applications.cs#L94> for a working example (= what the website does) or simply use `FaceAiSharp.Applications.CountEyeStates` yourself. Also see <https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/open-closed-eye-0001/README.md> for details about the model used, `open-closed-eye-0001`. The model page contains sample images too.
